spec/builtin-trap.test.sh

```diff
@@ -561,3 +561,63 @@ ok07
 trap-exit
 failure
 ## END
+
+#### trap '' sets handler to empty string (SIG_IGN)
```

Comment:
Hmm I think this test already passed before this change?

The problem is that `''` could mean either

- handle the signal, but run nothing
- don't handle with  SIG_IGN 

the second is the right behavior ...